client/src/components/Dashboard/AddItem.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState, useEffect } from 'react';
+import {
+  makeStyles,
+  MuiThemeProvider,
+  createMuiTheme,
+} from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Input from '@material-ui/core/Input';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import Typography from '@material-ui/core/Typography';
+
+const addItemTheme = createMuiTheme({
+  palette: {
+    primary: { main: '#DF1B1B' },
+  },
+});
```

Comment:
Since the primary palette is used throughout the entire project, it's best to move this setting into the themes/theme.js file. The theme.js already has this color set. Components can access the global themes by passing props into the useStyles() function.